src/ZoneServer/Skills/Handlers/Scout/DaggerSlash.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using static Melia.Zone.Skills.SkillUseFunctions;
+using System.Threading.Tasks;
+
+namespace Melia.Zone.Skills.Handlers.Scout
+{
+	/// <summary>
+	/// Handler for the Scout skill Dagger Slash.
+	/// </summary>
+	[SkillHandler(SkillId.Scout_DaggerSlash)]
+	public class DaggerSlash : IGroundSkillHandler
+	{
+		/// <summary>
+		/// Handles skill, do a slash attack to the nearby enemies.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		/// <param name="designatedTarget"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(designatedTarget);
```

Comment:
On my tests in game turning worked well without affecting the calculations, it seems to hit whatever was on the direction that he turned.

Reply:
That might be, but you didn't address my question^^ Is this actually what happens based on your logs? See, if my character looks straight up, and the designated target is at an angle of 10° from it, and it turns 10° towards the target, that means it will potentially hit different targets than it would've if the character hadn't turned. It's not a major issue, but a fine detail that has the potential to feel a bit wrong in-game.